### DISCUSSION TITLE
Why do people use digital coins instead of fiat money for payment?

### DISCUSSION THREAD

**Discussion by tktkowash** [JasmyCoin/JasmyCoin#19] [General]
We would like to discuss about the reason about using digital coins for payment. Please post as many reasons from various perspectives as possible.

**Comment by harayamada** [comment]
●The reason is...Digital token, which are not bound by national borders or conventional economic spheres, allow anyone to make payments freely, immediately, easily, and smoothly, and can be used for transactions around the world.

●The reason is …that there is a system in place that makes it more profitable and easy  to buy in tokens than in fiat money.
（ｍｍｍ・・・whats profitable？more cheeper？more benefits?
what you do give to users?


●the reason is...the producers will be paid properly,fairly。

**Comment by tktkowash** [comment]
From emotional aspect, there are some people who use a digital coin for payment just because it's innovative. They may want to have a superiority complex, want to feel as they are on the cutting edge, or just have a curiosity. A phrase "pay with Bitcoin" is very attractive for those people. Therefore, the more the digital coin gets to be common, the less they use it frequently.

**Reply by tktkowash** [reply]
One other thing where cryptocurrency is showing it's merits, is in use cases of cross-border settlements.  For example, if you create a data marketplace technology that enables people to trade data also between parties from different countries, settlement processes with traditional finance infrastructure will become more complicated. To do an international trade, you need a third party currency exchange involved for tradings. Cryptocurrency for "purpose" rather than "region" provides a framework for everyone to work together with the same underlying currency.

**Comment by tktkowash** [comment]
One of the interesting thing is that some digital coins have complete traceability. In case of donation, digital coins have a huge advantage over fiat money. If you donate with fiat money, it is almost impossible to trace who receive, how much they receive, and what it is used for. So, if there are two choices, donation with digital coins or with fiat money, is there any reason to use fiat money?

**Reply by harayamada** [reply]
Traceability is provided in theory, but the system needs to be adjusted to reflect real transparency based on the use case.   For example, if you want to use bitcoin for a charity foundation to create transparency, it may not be easy to actually achieve real transparency. This is because the bitcoin addresses are anonymous.  An evil charity foundation might find a way to transfer Bitcoin from one account to another account they also own, to pocket the money. If you do not know the owner of each account owner, you might not be able to achieve real transparency.

**Reply by tktkowash** [reply]
Yes, it's very important to identify what a certain account really is. I just suggest two ideas below:
1. Oracle
Trusted third party account verify who own a certain account. Of course, there still remain a problem "who verify oracle account?".
2. Transaction history
Based on past transaction history, a certain account can be verified whether it is really owned by charity foundation or not. If it has received/sent very few transactions, it may not be owned by charity foundation.

**Reply by harayamada** [reply]
Regarding transaction history, yes I believe you could make a blockchain network with a (KYC) mechanism where each public key is associated with a definite publicly identifiable party.   The names on the public addresses you see on etherscan are only displayed because a third party provides this service of displaying a name for registered wallets on their website. So it has actually nothing to do with the blockchain itself. Then the problem is that you have to trust that the registration was done correctly and under the right name

**Reply by tktkowash** [reply]
What we should trust for is not only registered name, but also transaction history itself. 

For example, imagine that some organizations publish their addresses on their web site:
Charity Foundation(0xCHR), UNICEF(0xUNI...), Red Cross(0xRED...), WFP(0xWFP...)

If there are many transactions from 0xCHR to 0xUNI, 0xRED and 0xWFP, address of 0xCHR is relatively trustworthy. Even if someone argue a certain address as new one of Charity Foundation, it is hard to be recognized as official address without transactions sent to organizations listed above. You can check those addresses either on their own website or on 3rd party website like Etherscan. If someone intend to deceive with a fake address, they have to override Charity Foundation's address first. After that, they also have to make as many transactions as 0xCHR has.
